cleanup/cleanup-bare-pods/cleanup-bare-pods.yaml

```diff
@@ -28,4 +28,4 @@ spec:
     - key: "{{ target.metadata.ownerReferences[] || `[]` }}"
       operator: Equals
       value: []
-  schedule: "*/5 * * * *"
+  schedule: "*/1 * * * *"
```

Comment:
Not sure if it's ok to update the original policy, I was thinking to replicate the policy in the test folder and update the schedule.

@chipzoller - is this policy used anywhere else?